tests/integration/conftest.py

```diff
@@ -104,3 +107,23 @@ def inner():
         return ws_group, acc_group
 
     return inner
+
+
+class StaticTablesCrawler(TablesCrawler):
```

Comment:
I'm confused, why do we introduce a specific Table Crawler instead of modifying the existing one ? This way, we'll be able to run UCX on a very small scope, running the assessment + group migration + data migration for one table / catalog

Reply:
it's specifically added for integration tests, following the same interface, so that it can be swapped